src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="content">
+    <section class="calend content__calend calend--start-day--mon calend--month-length--31">
+      <article class="calend__day calend__day--date--1"></article>
```

Comment:
```suggestion
      <article class="calend__day"></article>
```
do not use day numbers here, use loop in scss to add numbers